CHANGELOG.md

```diff
@@ -8,7 +8,8 @@ x.x.x Release notes (yyyy-MM-dd)
 
 ### Enhancements
 
-* None.
+* Swift: Added `List.invalidated`, which returns if the given `List` is no longer
```

Comment:
Well, `List.invalidated` returns the curried function, so I disagree. I'd use `List.Self.foo` or `List.Type.foo` for type-level functions.